CONTRIBUTING.md

```diff
@@ -6,7 +6,7 @@ We'd love to accept your sample apps and patches! Before we can take them, we
 have to jump a couple of legal hurdles.
 
 Please fill out either the individual or corporate Contributor License Agreement
-(CLA).
+(CLA). For more information on the purpose of CLA, see https://en.wikipedia.org/wiki/Contributor_License_Agreement
```

Comment:
How about see [this](https://en.wikipedia.org/wiki/Contributor_License_Agreement)

Remove spaces between between square and round brackets in my suggestion for better effect. I noticed that links after are also weird formatted due to extra space between closing square bracket and opening round brackets. I think you can fix it ;)

--
Disclaimer: I am not owner of this repository or Google team member